webui/react/src/components/HyperparameterSearchModal.test.tsx

```diff
@@ -132,7 +132,7 @@ describe('useModalHyperparameterSearch', () => {
     const { view } = await setup();
 
     await user.click(view.getByRole('button', { name: 'Select Hyperparameters' }));
-    await user.click(view.getByRole('button', { name: 'Run Experiment' }));
+    await user.click(view.getByRole('button', { name: 'Run Search' }));
```

Comment:
nit: in order to work whether the feature flag is on or off, this could be `view.getByRole('button', { name: 'Run' , exact: false})`